docs/source/reference/server-api.md

```diff
@@ -119,67 +101,50 @@ while the server is not ready yet:
   }
 ```
 
-Note that `ready` is false and `pending` is `spawn`.
-This means that the server is not ready
-(attempting to access it may not work)
-because it isn't finished spawning yet.
-We'll get more into that below in [waiting for a server][].
+Note that `ready` is assigned `false` and `pending` is assigned `spawn`, meaning that the server is not ready and attempting to access it may not work as it is still in the process of spawning.
```

Comment:
I'm not sure `assigned` is helpful here. I would still say `ready is false`. But if you do find we need different language, I would probably pick something like 'has the value'. But in short, I think `is` is clear and concise and standard language for a field having a value.

Can you speak more to removing the link? It seems useful to keep it here that we're going to get more into that later.

Reply:
I have removed `assigned` as suggested. I have also returned the link as your explanation makes sense. I removed it as I figured the reader will get to the section either way, so it did not add any value having the information.